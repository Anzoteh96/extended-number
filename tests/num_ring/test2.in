-829976
421523
-709524
