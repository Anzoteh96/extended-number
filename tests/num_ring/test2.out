Enter the number base b:
Enter two numbers n and m:
b is 829976
n mod b is 421523
m mod b is 120452
n + m is 541975
n - m is 301071
n * m is 336572
n ** m is 25601
